### ISSUE TITLE
2025-04-14, Papers for large language model, llm

### ISSUE THREAD

**Issue by 9tailwolf** [Arxiv-Sender/Arxiv-Sender#140]
# Today's Arxiv Papers

(1) [Quantum Large Language Model Fine-Tuning](http://arxiv.org/abs/2504.08732v1), 2025-04-11

(2) [Visual Chronicles: Using Multimodal LLMs to Analyze Massive Collections of Images](http://arxiv.org/abs/2504.08727v1), 2025-04-11

(3) [DocAgent: A Multi-Agent System for Automated Code Documentation Generation](http://arxiv.org/abs/2504.08725v1), 2025-04-11

(4) [Generating Fine Details of Entity Interactions](http://arxiv.org/abs/2504.08714v1), 2025-04-11

(5) [SWE-PolyBench: A multi-language benchmark for repository level evaluation of coding agents](http://arxiv.org/abs/2504.08703v1), 2025-04-11

(6) [Large Language Models as Span Annotators](http://arxiv.org/abs/2504.08697v1), 2025-04-11

(7) [SeaView: Software Engineering Agent Visual Interface for Enhanced Workflow](http://arxiv.org/abs/2504.08696v1), 2025-04-11

(8) [TP-RAG: Benchmarking Retrieval-Augmented Large Language Model Agents for Spatiotemporal-Aware Travel Planning](http://arxiv.org/abs/2504.08694v1), 2025-04-11

(9) [Fast-Slow-Thinking: Complex Task Solving with Large Language Models](http://arxiv.org/abs/2504.08690v1), 2025-04-11

(10) [Voice Interaction With Conversational AI Could Facilitate Thoughtful Reflection and Substantive Revision in Writing](http://arxiv.org/abs/2504.08687v1), 2025-04-11

(11) [Genius: A Generalizable and Purely Unsupervised Self-Training Framework For Advanced Reasoning](http://arxiv.org/abs/2504.08672v1), 2025-04-11

(12) [Variability-Driven User-Story Generation using LLM and Triadic Concept Analysis](http://dx.doi.org/10.5220/0013360500003928), 2025-04-11

(13) [Quality evaluation of Tabby coding assistant using real source code snippets](http://arxiv.org/abs/2504.08650v1), 2025-04-11

(14) [Do LLMs trust AI regulation? Emerging behaviour of game-theoretic LLM agents](http://arxiv.org/abs/2504.08640v1), 2025-04-11

(15) [MooseAgent: A LLM Based Multi-agent Framework for Automating Moose Simulation](http://arxiv.org/abs/2504.08621v1), 2025-04-11

(16) [Analyzing 16,193 LLM Papers for Fun and Profits](http://arxiv.org/abs/2504.08619v1), 2025-04-11

(17) [Playpen: An Environment for Exploring Learning Through Conversational Interaction](http://arxiv.org/abs/2504.08590v1), 2025-04-11

(18) [UoB-NLP at SemEval-2025 Task 11: Leveraging Adapters for Multilingual and Cross-Lingual Emotion Detection](http://arxiv.org/abs/2504.08543v1), 2025-04-11

(19) [Embodied Image Captioning: Self-supervised Learning Agents for Spatially Coherent Image Descriptions](http://arxiv.org/abs/2504.08531v1), 2025-04-11

(20) [Task Memory Engine (TME): Enhancing State Awareness for Multi-Step LLM Agent Tasks](http://arxiv.org/abs/2504.08525v1), 2025-04-11

(21) [Adopting Large Language Models to Automated System Integration](http://arxiv.org/abs/2504.08490v1), 2025-04-11

(22) [TickIt: Leveraging Large Language Models for Automated Ticket Escalation](http://dx.doi.org/10.1145/3696630.3728558), 2025-04-11

(23) [Ego4o: Egocentric Human Motion Capture and Understanding from Multi-Modal Input](http://arxiv.org/abs/2504.08449v1), 2025-04-11

(24) [A Reproducibility Study of Graph-Based Legal Case Retrieval](http://arxiv.org/abs/2504.08400v1), 2025-04-11

(25) [Beyond Self-Reports: Multi-Observer Agents for Personality Assessment in Large Language Models](http://arxiv.org/abs/2504.08399v1), 2025-04-11

(26) [PCA-RAG: Principal Component Analysis for Efficient Retrieval-Augmented Generation](http://arxiv.org/abs/2504.08386v1), 2025-04-11

(27) [Scaling Up On-Device LLMs via Active-Weight Swapping Between DRAM and Flash](http://arxiv.org/abs/2504.08378v1), 2025-04-11

(28) [MedRep: Medical Concept Representation for General Electronic Health Record Foundation Models](http://arxiv.org/abs/2504.08329v1), 2025-04-11

(29) [SortBench: Benchmarking LLMs based on their ability to sort lists](http://arxiv.org/abs/2504.08312v1), 2025-04-11

(30) [DSM: Building A Diverse Semantic Map for 3D Visual Grounding](http://arxiv.org/abs/2504.08307v1), 2025-04-11

(31) [Large language models could be rote learners](http://arxiv.org/abs/2504.08300v1), 2025-04-11

(32) [ELSA: A Style Aligned Dataset for Emotionally Intelligent Language Generation](http://arxiv.org/abs/2504.08281v1), 2025-04-11

(33) [Evaluating the Bias in LLMs for Surveying Opinion and Decision Making in Healthcare](http://arxiv.org/abs/2504.08260v1), 2025-04-11

(34) [RAG-VR: Leveraging Retrieval-Augmented Generation for 3D Question Answering in VR Environments](http://arxiv.org/abs/2504.08256v1), 2025-04-11

(35) [Jupiter: Fast and Resource-Efficient Collaborative Inference of Generative LLMs on Edge Devices](http://arxiv.org/abs/2504.08242v1), 2025-04-11

(36) [Out of Style: RAG's Fragility to Linguistic Variation](http://arxiv.org/abs/2504.08231v1), 2025-04-11

(37) [F$^3$Set: Towards Analyzing Fast, Frequent, and Fine-grained Events from Videos](http://arxiv.org/abs/2504.08222v1), 2025-04-11

(38) [LLM for Comparative Narrative Analysis](http://arxiv.org/abs/2504.08211v1), 2025-04-11

(39) [How Good Are Large Language Models for Course Recommendation in MOOCs?](http://arxiv.org/abs/2504.08208v1), 2025-04-11

(40) [DRAFT-ing Architectural Design Decisions using LLMs](http://arxiv.org/abs/2504.08207v1), 2025-04-11

(41) [Harnessing the Unseen: The Hidden Influence of Intrinsic Knowledge in Long-Context Language Models](http://arxiv.org/abs/2504.08202v1), 2025-04-11

(42) [SAEs $\textit{Can}$ Improve Unlearning: Dynamic Sparse Autoencoder Guardrails for Precision Unlearning in LLMs](http://arxiv.org/abs/2504.08192v1), 2025-04-11

(43) [A Vulnerability Code Intent Summary Dataset](http://arxiv.org/abs/2504.08180v1), 2025-04-11

(44) [GenXSS: an AI-Driven Framework for Automated Detection of XSS Attacks in WAFs](http://arxiv.org/abs/2504.08176v1), 2025-04-11

(45) [Findings of the BabyLM Challenge: Sample-Efficient Pretraining on Developmentally Plausible Corpora](http://dx.doi.org/10.18653/v1/2023.conll-babylm.1), 2025-04-10

(46) [Investigating Vision-Language Model for Point Cloud-based Vehicle Classification](http://arxiv.org/abs/2504.08154v1), 2025-04-10

(47) [Orchestrating Agents and Data for Enterprise: A Blueprint Architecture for Compound AI](http://arxiv.org/abs/2504.08148v1), 2025-04-10

(48) [Empowering Vector Architectures for ML: The CAMP Architecture for Matrix Multiplication](http://arxiv.org/abs/2504.08137v1), 2025-04-10

(49) [Gen3DEval: Using vLLMs for Automatic Evaluation of Generated 3D Objects](http://arxiv.org/abs/2504.08125v1), 2025-04-10

(50) [DeepSeek vs. o3-mini: How Well can Reasoning LLMs Evaluate MT and Summarization?](http://arxiv.org/abs/2504.08120v1), 2025-04-10

(51) [Test Amplification for REST APIs via Single and Multi-Agent LLM Systems](http://arxiv.org/abs/2504.08113v1), 2025-04-10

(52) [Scaling Laws of Graph Neural Networks for Atomistic Materials Modeling](http://arxiv.org/abs/2504.08112v1), 2025-04-10

(53) [POEM: Precise Object-level Editing via MLLM control](http://arxiv.org/abs/2504.08111v1), 2025-04-10

(54) [Geneshift: Impact of different scenario shift on Jailbreaking LLM](http://arxiv.org/abs/2504.08104v1), 2025-04-10

(55) [Can Reasoning LLMs Enhance Clinical Document Classification?](http://arxiv.org/abs/2504.08040v1), 2025-04-10

(56) [C3PO: Critical-Layer, Core-Expert, Collaborative Pathway Optimization for Test-Time Expert Re-Mixing](http://arxiv.org/abs/2504.07964v1), 2025-04-10

(57) [GLUS: Global-Local Reasoning Unified into A Single Large Language Model for Video Segmentation](http://arxiv.org/abs/2504.07962v1), 2025-04-10

(58) [MM-IFEngine: Towards Multimodal Instruction Following](http://arxiv.org/abs/2504.07957v1), 2025-04-10

(59) [VCR-Bench: A Comprehensive Evaluation Framework for Video Chain-of-Thought Reasoning](http://arxiv.org/abs/2504.07956v1), 2025-04-10

(60) [Scaling Laws for Native Multimodal Models](http://arxiv.org/abs/2504.07951v2), 2025-04-10

(61) [Porting an LLM based Application from ChatGPT to an On-Premise Environment](http://arxiv.org/abs/2504.07907v1), 2025-04-10

(62) [Redefining Machine Translation on Social Network Services with Large Language Models](http://arxiv.org/abs/2504.07901v1), 2025-04-10

(63) [How do Large Language Models Understand Relevance? A Mechanistic Interpretability Perspective](http://arxiv.org/abs/2504.07898v1), 2025-04-10

(64) [Benchmarking Adversarial Robustness to Bias Elicitation in Large Language Models: Scalable Automated Assessment with LLM-as-a-Judge](http://arxiv.org/abs/2504.07887v1), 2025-04-10

(65) [An LLM-Driven Multi-Agent Debate System for Mendelian Diseases](http://arxiv.org/abs/2504.07881v2), 2025-04-10

(66) [Token Level Routing Inference System for Edge Devices](http://arxiv.org/abs/2504.07878v1), 2025-04-10

(67) [Pangu Ultra: Pushing the Limits of Dense Large Language Models on Ascend NPUs](http://arxiv.org/abs/2504.07866v2), 2025-04-10

(68) [Robust Hallucination Detection in LLMs via Adaptive Token Selection](http://arxiv.org/abs/2504.07863v1), 2025-04-10

(69) [Emergence of psychopathological computations in large language models](http://arxiv.org/abs/2504.08016v1), 2025-04-10

(70) [2D-Curri-DPO: Two-Dimensional Curriculum Learning for Direct Preference Optimization](http://arxiv.org/abs/2504.07856v1), 2025-04-10

(71) [The KL3M Data Project: Copyright-Clean Training Resources for Large Language Models](http://arxiv.org/abs/2504.07854v1), 2025-04-10

(72) [Understanding Learner-LLM Chatbot Interactions and the Impact of Prompting Guidelines](http://arxiv.org/abs/2504.07840v1), 2025-04-10

(73) [Deceptive Automated Interpretability: Language Models Coordinating to Fool Oversight Systems](http://arxiv.org/abs/2504.07831v1), 2025-04-10

(74) [MOSAIC: Modeling Social AI for Content Dissemination and Regulation in Multi-Agent Simulations](http://arxiv.org/abs/2504.07830v1), 2025-04-10

(75) [Cluster-Driven Expert Pruning for Mixture-of-Experts Large Language Models](http://arxiv.org/abs/2504.07807v1), 2025-04-10

(76) [A System for Comprehensive Assessment of RAG Frameworks](http://arxiv.org/abs/2504.07803v1), 2025-04-10

(77) [FairEval: Evaluating Fairness in LLM-Based Recommendations with Personality Awareness](http://arxiv.org/abs/2504.07801v1), 2025-04-10

(78) [Plan-and-Refine: Diverse and Comprehensive Retrieval-Augmented Generation](http://arxiv.org/abs/2504.07794v1), 2025-04-10

(79) [Fairness Mediator: Neutralize Stereotype Associations to Mitigate Bias in Large Language Models](http://dx.doi.org/10.1145/3728881), 2025-04-10

(80) [Efficient Tuning of Large Language Models for Knowledge-Grounded Dialogue Generation](http://arxiv.org/abs/2504.07754v1), 2025-04-10

(81) [SF2T: Self-supervised Fragment Finetuning of Video-LLMs for Fine-Grained Understanding](http://arxiv.org/abs/2504.07745v1), 2025-04-10

(82) [Zero-Shot Cross-Domain Code Search without Fine-Tuning](http://dx.doi.org/10.1145/3729357), 2025-04-10

(83) [Automated Construction of a Knowledge Graph of Nuclear Fusion Energy for Effective Elicitation and Retrieval of Information](http://arxiv.org/abs/2504.07738v1), 2025-04-10

(84) [DeepGreen: Effective LLM-Driven Green-washing Monitoring System Designed for Empirical Testing -- Evidence from China](http://arxiv.org/abs/2504.07733v1), 2025-04-10

(85) [MRD-RAG: Enhancing Medical Diagnosis with Multi-Round Retrieval-Augmented Generation](http://arxiv.org/abs/2504.07724v1), 2025-04-10

(86) [PR-Attack: Coordinated Prompt-RAG Attacks on Retrieval-Augmented Generation in Large Language Models via Bilevel Optimization](http://arxiv.org/abs/2504.07717v1), 2025-04-10

(87) [Proactive User Information Acquisition via Chats on User-Favored Topics](http://arxiv.org/abs/2504.07698v1), 2025-04-10

(88) [FMNV: A Dataset of Media-Published News Videos for Fake News Detection](http://arxiv.org/abs/2504.07687v1), 2025-04-10

(89) [Synthetic Fluency: Hallucinations, Confabulations, and the Creation of Irish Words in LLM-Generated Translations](http://arxiv.org/abs/2504.07680v1), 2025-04-10

(90) [Unveiling the Impact of Multimodal Features on Chinese Spelling Correction: From Analysis to Design](http://arxiv.org/abs/2504.07661v1), 2025-04-10

(91) [Synthesizing High-Quality Programming Tasks with LLM-based Expert and Student Agents](http://arxiv.org/abs/2504.07655v1), 2025-04-10

(92) [On the Temporal Question-Answering Capabilities of Large Language Models Over Anonymized Data](http://arxiv.org/abs/2504.07646v1), 2025-04-10

(93) [Enhancing Large Language Models through Neuro-Symbolic Integration and Ontological Reasoning](http://arxiv.org/abs/2504.07640v1), 2025-04-10

(94) [Agent That Debugs: Dynamic State-Guided Vulnerability Repair](http://arxiv.org/abs/2504.07634v1), 2025-04-10

(95) [ConceptFormer: Towards Efficient Use of Knowledge-Graph Embeddings in Large Language Models](http://arxiv.org/abs/2504.07624v1), 2025-04-10

(96) [VLM-R1: A Stable and Generalizable R1-style Large Vision-Language Model](http://arxiv.org/abs/2504.07615v1), 2025-04-10

(97) [Boosting Universal LLM Reward Design through Heuristic Reward Observation Space Evolution](http://arxiv.org/abs/2504.07596v2), 2025-04-10

(98) [REANIMATOR: Reanimate Retrieval Test Collections with Extracted and Synthetic Resources](http://arxiv.org/abs/2504.07584v1), 2025-04-10

(99) [Do LLMs Understand Your Translations? Evaluating Paragraph-level MT with Question Answering](http://arxiv.org/abs/2504.07583v2), 2025-04-10

(100) [Malware analysis assisted by AI with R2AI](http://arxiv.org/abs/2504.07574v2), 2025-04-10
